modules/common/react/lib/utils/changeToStaticStates.ts

```diff
@@ -0,0 +1,62 @@
+import {CSSObject, Interpolation} from '@emotion/core';
+import {WithTheme} from '@emotion/styled';
+import {Overwrapped} from '@emotion/styled-base/types/helper';
+
+/**
+ * Allows the ability to force a state by a pseudo selector by changing the selector into a class creating a new styled object with the class names
+ * @param shouldChangePseudoSelectors If true, it will replace pseudo selectors from the styled object and change them to classes
+ * @param obj The styled object passed to the function to replace any pseudo selectors with a class name
+ */
+
+export interface ChangeStaticStateProps {
+  /** @ignore */
+  shouldChangeStyleToStaticStates?: boolean;
```

Comment:
I wonder if this should be prefixed so that if anyone sees it in the codebase, it's immediately clear that it's an internal prop. I feel less bad if we make any breaking changes to it in that case haha.

Maybe something as simple as an underscore (`_shouldConvertPseudoStatesToClasses`)?

Reply:
since it's such an internal thing, I'm ok with this!